### DISCUSSION TITLE
PURCHASEGIFT problem

### DISCUSSION THREAD

**Discussion by javagrabber** [chr233/ASFEnhance#89] [提问求助 / Support]
There is a problem when transferring gift from botA to botB.
![ошибка](https://user-images.githubusercontent.com/125265529/222075131-04d03147-fd40-49c0-a85f-8bf815887d30.jpg)
botA and botB belong to the same region (Argentina). When you enter a botB account through the site, the gift is accepted in manual mode normally. I can't understand where is the error?

**Comment by chr233** [comment]
lol, I cant offer help for this, this error message is from steam web API, maybe you can use `balance asf` to check these bots' balance currency

**Comment by javagrabber** [comment]
Balance botA is fine. The botA buys gifts and gives them to the botB. botB doesn't accept them. I need to go to the botB account and accept manually

**Comment by chr233** [comment]
ASF has a configuration to let bot accept gift automaticly, `AcceptGifts`

**Reply by javagrabber** [reply]
I know this. The bot accepts gifts from skins and trading cards perfectly. feature is enabled. This issue only occurs when giving away games as a gift.

**Reply by chr233** [reply]
do you set `AcceptGifts` = `true` ?

**Reply by javagrabber** [reply]
Yes. on all of my bots

**Reply by javagrabber** [reply]
Fix it. Need to use VPN or proxy of the country in which the account receiving the games as a gift is registered. Thanks to all!
